src/main/target/JHEF_JHEF7DUAL/target.mk

```diff
@@ -0,0 +1,18 @@
+F7X5XG_TARGETS += $(TARGET)
```

Comment:
F7X2RE_TARGETS += $(TARGET)

Reply:
thanks. i need to fix my converter script to match.

Reply:
i'm thinking this only changes an environment variable for the `make` command for when a user wants to build a subset of targets by chip-type.